packages/vscode/package.json

```diff
@@ -88,10 +88,37 @@
           ],
           "default": null,
           "description": "Specifies the path to a GameMaker project to use as a template to clone when creating a new project via the explorer context menu. Defaults to a simple project with a single room and a single object."
+        },
+        "stitch.gmlSpec.source": {
+          "title": "GameMaker GmlSpec.xml file source",
+          "type": [
+            "string"
+          ],
+          "enum": [
+            "localRuntime",
+            "external",
+            "internal"
+          ],
+          "default": "localRuntime",
```

Comment:
I like where you're going with this, but the implementation doesn't account for the various possible GameMaker versions (beta, stable, LTS). I'll change the default to "internal" for now to accommodate that, but I like the idea of having it default to local once the implementation accounts for all the various local scenarios we can end up with.